osu.Game.Rulesets.Osu.Tests/TestSceneSpinnerInput.cs

```diff
@@ -194,11 +196,11 @@ public void TestRewind()
             assertTotalRotation(3750, 810);
             assertTotalRotation(3500, 720);
             assertTotalRotation(3250, 530);
```

Comment:
I'm fairly certain this is wrong as well but is saved by the test lenience.

It's hard to assert what this value should actually be since it's not explicitly defined, but... If this sequence is a 450deg delta over 500ms, then at 3250ms we have half of that (225deg). Since the rotation starts at -90deg, then this should land at `-90 + 225 = 135`. But, since we're in a segment that has reached -180deg, this is still dominated by `360 + 180 = 540`.

So I believe the correct assertion here is `540 + 135 = 675`

Reply:
This is primarily pointing out that, Strictly speaking, _all of these_ should be decreasing as time is rewound.